Makefile

```diff
@@ -23,12 +23,13 @@ src/python_pachyderm/proto: docker-build-proto
 	cd proto/pachyderm && \
 		git fetch --all && \
 		git checkout v$(PACHYDERM_VERSION)
-	find ./proto/pachyderm/src/client -regex ".*\.proto" \
+	find ./proto/pachyderm/ -regex ".*\.proto" \
 	| grep -v 'internal' \
 	| xargs tar cf - \
 	| docker run -i pachyderm_python_proto \
 	| tar xf -
-	mv src/python_pachyderm/client src/python_pachyderm/proto
+	rm -rf src/python_pachyderm/proto
```

Comment:
Redundant line, since we already have one above
https://github.com/pachyderm/python-pachyderm/pull/260/files?file-filters%5B%5D=No+extension#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L22